README.md

```diff
@@ -354,9 +354,8 @@ These are the available config options for making requests. Only the `url` is re
   maxContentLength: 2000,
 
   // `validateStatus` defines whether to resolve or reject the promise for a given
-  // HTTP response status code. If `validateStatus` returns `true` (or is set to `null`
-  // or `undefined`), the promise will be resolved; otherwise, the promise will be
-  // rejected.
+  // HTTP response status code. If `validateStatus` returns `true` (or is set to `null`), 
+  // the promise will be resolved; otherwise, the promise will be rejected.
```

Comment:
this docs were sort of wrong from the looks of it, the functionallity checks if `validateStatus` is passed, not if it's specifically `null`, it would be better to update this part